rudder-core/src/main/scala/com/normation/rudder/services/policies/write/BuildBundleSequence.scala

```diff
@@ -259,7 +256,15 @@ class BuildBundleSequence(
       case Some(cfg) =>
         val techniqueBundles = cfg.bundlesequence.map { bundleName =>
           if(bundleName.value.trim.size > 0) {
-            List(Bundle(ReportId(policy.id.getReportId), bundleName))
+            val vars = if (policy.technique.generationMode == TechniqueGenerationMode.MultipleDirectivesWithParameters)
```

Comment:
could you add { } around the for, to improve readability ? I had trouble parsing that the for what linked to the if